llvm/test/CodeGen/AIE/aie2/schedule/mov_scl.mir

```diff
@@ -0,0 +1,270 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+# RUN: llc -march=aie2 -run-pass=postmisched %s %topdown-multi -o - | FileCheck %s
```

Comment:
nit: better to use mtriple instead, in this way we can select target+march, not just march considering the default target.

Reply:
And I think I heard `-march` will get deprecated